front/components/assistant/conversation/input_bar/InputBarContainer.tsx

```diff
@@ -26,6 +26,8 @@ export const INPUT_BAR_ACTIONS = ["attachment", "quick-actions"] as const;
 
 export type InputBarAction = (typeof INPUT_BAR_ACTIONS)[number];
 
+const supportedFileExtensions = [".txt", ".csv", ".tsv", ".md", ".pdf"];
```

Comment:
Why adding `.tsv`? Can we ensure that this PR is only about refactoring and does not improve coverage of content type?